app/main/views/drafts.py

```diff
@@ -298,6 +306,41 @@ def create_new_draft_service(framework_slug):
     return jsonify(services=draft.serialize()), 201
 
 
+@main.route('/draft-services/<int:draft_id>/complete', methods=['POST'])
+def complete_draft_service(draft_id):
+    updater_json = validate_and_return_updater_request()
+
+    draft = DraftService.query.filter(
+        DraftService.id == draft_id
+    ).first_or_404()
+
+    errs = validate_draft(draft)
+    if errs:
+        abort(400, errs)
+
+    draft.status = 'submitted'
+    try:
+        db.session.add(draft)
+        db.session.flush()
+
+        audit = AuditEvent(
+            audit_type=AuditTypes.complete_draft_service,
+            user=updater_json['updated_by'],
+            data={
+                "draftId": draft.id
```

Comment:
I'm not really sure, I think this started when we thought that `draftId` might be `service_id`, but I kept it here because all other draft events write `draftId`.